safenode/src/domain/storage/disk_backed_record_store.rs

```diff
@@ -246,19 +246,22 @@ impl RecordStore for DiskBackedRecordStore {
     }
 
     fn add_provider(&mut self, _record: ProviderRecord) -> Result<()> {
-        todo!("ProviderRecords are not used currently.")
+        // ProviderRecords are not used currently
+        Ok(())
     }
 
     fn providers(&self, _key: &Key) -> Vec<ProviderRecord> {
-        todo!("ProviderRecords are not used currently.")
+        // ProviderRecords are not used currently
+        vec![]
     }
 
     fn provided(&self) -> Self::ProvidedIter<'_> {
-        todo!("ProviderRecords are not used currently.")
+        // ProviderRecords are not used currently
```

Comment:
better change it to `warn` log, so that we can trace the kad behaviour if interested?

Reply:
i suspect this will flood the logs as these methods are called periodically by kad...not sure tbh

Reply:
it shall not. 
if does, the current `todo` shall already cause trouble.